### PR TITLE
INS-1454: Rewrite logs.

### PR DESCRIPTION
**Pull request by andreyromancev**

### CODE REVIEW

File: ledger/pulsemanager/pulsemanager.go [hunk 1]
```diff
@@ -448,15 +437,21 @@ func (m *PulseManager) processJets(ctx context.Context, currentPulse, newPulse c
...
 		executor, err := m.JetCoordinator.LightExecutorForJet(ctx, jetID, currentPulse)
 		if err != nil {
 			return nil, err
 		}
 		imExecutor := *executor == me
-		logger.Debugf("[jet]: %v process. Pulse: %v, Executor: %v", jetID.DebugString(), currentPulse, imExecutor)
+		logger.WithFields(map[string]interface{}{
+			"jet":      jetID.DebugString(),
+			"executor": imExecutor,
+		}).Debug("process jet")
```
Comment (ruz) [inline ledger/pulsemanager/pulsemanager.go]: this is very wrong!!!!!

File: ledger/artifactmanager/artifactmanager_senders.go [hunk 2]
```diff
@@ -96,8 +92,6 @@ func followRedirectSender(bus core.MessageBus) PreSender {
 				stats.Record(ctx, statRedirects.M(1))
 
 				redirected := r.Redirected(msg)
-				inslog.Debugf("redirect reciever=%v", r.GetReceiver())
```
Comment (nordicdyno) [inline ledger/artifactmanager/artifactmanager_senders.go]: `+`

File: ledger/artifactmanager/handler.go [hunk 3]
```diff
@@ -404,11 +397,6 @@ func (h *MessageHandler) handleGetObject(
 			return nil, fmt.Errorf("failed to fetch index for %s", msg.Head.Record().String())
 		}
 
-		logger.Errorf(
```
Comment (nordicdyno) [inline ledger/artifactmanager/handler.go]: debug

File: ledger/artifactmanager/handler.go [hunk 4]
```diff
@@ -450,13 +438,6 @@ func (h *MessageHandler) handleGetObject(
 			if err != nil {
 				return nil, err
 			}
-			logger.Debugf(
```
Comment (nordicdyno) [inline ledger/artifactmanager/handler.go]: `+`

File: ledger/artifactmanager/handler.go [hunk 5]
```diff
@@ -486,14 +467,6 @@ func (h *MessageHandler) handleGetObject(
 		if err != nil {
 			return nil, err
 		}
-
-		logger.Debugf(
```
Comment (nordicdyno) [inline ledger/artifactmanager/handler.go]: `+`

File: ledger/artifactmanager/handler.go [hunk 6]
```diff
@@ -811,15 +769,13 @@ func (h *MessageHandler) handleUpdateObject(ctx context.Context, parcel core.Par
...
 		var err error
-		logger.Debugf("Get index for: %v, jet: %v", msg.Object.Record(), jetID.DebugString())
 		idx, err = tx.GetObjectIndex(ctx, jetID, msg.Object.Record(), true)
 		// No index on our node.
 		if err == storage.ErrNotFound {
 			if state.State() == record.StateActivation {
 				// We are activating the object. There is no index for it anywhere.
 				idx = &index.ObjectLifeline{State: record.StateUndefined}
 			} else {
-				logger.Debugf("Not found index for: %v, jet: %v", msg.Object.Record(), jetID.DebugString())
```
Comment (nordicdyno) [inline ledger/artifactmanager/handler.go]: better description

File: ledger/artifactmanager/hotdatawaiter.go [hunk 7]
```diff
@@ -75,42 +70,32 @@ func (hdw *HotDataWaiterConcrete) getWaiter(ctx context.Context, jetID core.Reco
...
 // ThrowTimeout raises all timeoutChannel
 func (hdw *HotDataWaiterConcrete) ThrowTimeout(ctx context.Context) {
-	logger := inslogger.FromContext(ctx)
-	logger.Debugf("[ThrowTimeout] start method")
-
 	hdw.waitersMapLock.Lock()
 	defer hdw.waitersMapLock.Unlock()
 
-	for jetID, waiter := range hdw.waiters {
-		logger.Debugf("[ThrowTimeout] raising timeout for requests with jetID - %v", jetID.DebugString())
```
Comment (nordicdyno) [inline ledger/artifactmanager/hotdatawaiter.go]: `+`

File: ledger/artifactmanager/instrument.go [hunk 8]
```diff
@@ -57,8 +57,6 @@ func (mi *methodInstrumenter) end() {
 		inslog.Error(*mi.errlink)
 	}
 
-	inslog.Debugf("measured time of AM method %v is %v", mi.name, latency)
```
Comment (nordicdyno) [inline ledger/artifactmanager/instrument.go]: `+`

File: ledger/artifactmanager/jettreeupdater.go [hunk 9]
```diff
@@ -74,18 +74,10 @@ func (jtu *jetTreeUpdater) fetchJet(
...
 	if actual {
-		inslogger.FromContext(ctx).Debugf(
-			"we believe object %s is in JET %s", target.String(), jetID.DebugString(),
-		)
 		return jetID, nil
 	}
 
 	span.Annotate(nil, "tree in DB is not actual")
-	inslogger.FromContext(ctx).Debugf(
-		"jet %s is not actual in our tree, asking neighbors for jet of object %s",
```
Comment (nordicdyno) [inline ledger/artifactmanager/jettreeupdater.go]: convert to comment

File: ledger/artifactmanager/jettreeupdater.go [hunk 10]
```diff
@@ -104,20 +96,13 @@ func (jtu *jetTreeUpdater) fetchJet(
 	if mu.done {
 		mu.Unlock()
 		span.Annotate(nil, "somebody else updated actuality")
-		inslogger.FromContext(ctx).Debugf(
-			"somebody else updated actuality of jet %s, rechecking our DB",
```
Comment (nordicdyno) [inline ledger/artifactmanager/jettreeupdater.go]: convert to comment

File: ledger/exporter/exporter.go [hunk 11]
```diff
@@ -78,12 +77,9 @@ type pulseData struct {
 // Export returns data view from storage.
 func (e *Exporter) Export(ctx context.Context, fromPulse core.PulseNumber, size int) (*core.StorageExportResult, error) {
 	result := core.StorageExportResult{Data: map[string]interface{}{}}
-	inslog := inslogger.FromContext(ctx)
-	inslog.Debugf("[ API Export ] start")
 
 	jetIDs, err := e.JetStorage.GetJets(ctx)
 	if err != nil {
-		inslog.Debugf("[ API Export ] error getting jets: %s", err.Error())
 		return nil, err
```
Comment (nordicdyno) [inline ledger/exporter/exporter.go]: annotate error

File: ledger/heavyclient/heavy_client.go [hunk 12]
```diff
@@ -205,14 +205,11 @@ func (c *JetClient) syncloop(ctx context.Context) {
...
 			syncPN, hasNext = c.nextPulseNumber()
 			if hasNext {
-				inslog.Debugf("synchronization next sync pulse num: %v (left=%v)", syncPN, c.leftPulses)
 				break
 			}
 
-			inslog.Debug("synchronization waiting signal what new pulse happens")
 			_, ok := <-c.signal
 			if !ok {
-				inslog.Debug("stop is called, so we are should just stop syncronization loop")
```
Comment (nordicdyno) [inline ledger/heavyclient/heavy_client.go]: Info

File: ledger/heavyclient/heavy_client.go [hunk 13]
```diff
@@ -205,14 +205,11 @@ func (c *JetClient) syncloop(ctx context.Context) {
 			// if we have pulses to sync, process it
 			syncPN, hasNext = c.nextPulseNumber()
 			if hasNext {
-				inslog.Debugf("synchronization next sync pulse num: %v (left=%v)", syncPN, c.leftPulses)
 				break
 			}
 
-			inslog.Debug("synchronization waiting signal what new pulse happens")
```
Comment (nordicdyno) [inline ledger/heavyclient/heavy_client.go]: `+`

File: ledger/heavyclient/heavy_client.go [hunk 14]
```diff
@@ -205,14 +205,11 @@ func (c *JetClient) syncloop(ctx context.Context) {
 			// if we have pulses to sync, process it
 			syncPN, hasNext = c.nextPulseNumber()
 			if hasNext {
-				inslog.Debugf("synchronization next sync pulse num: %v (left=%v)", syncPN, c.leftPulses)
```
Comment (nordicdyno) [inline ledger/heavyclient/heavy_client.go]: `+`

File: ledger/heavyclient/heavy_pool.go [hunk 15]
```diff
@@ -196,10 +196,6 @@ func (scp *Pool) LightCleanup(
 					<-sem
 				}()
 				_, _, _ = scp.cleanupGroup.Do(string(jetPrefix), func() (interface{}, error) {
-
-					inslogger.FromContext(ctx).Debugf("Start light cleanup, pulse < %v, jet = %v",
```
Comment (nordicdyno) [inline ledger/heavyclient/heavy_pool.go]: `+`

File: ledger/pulsemanager/pulsemanager.go [hunk 16]
```diff
@@ -504,18 +498,16 @@ func (m *PulseManager) processJets(ctx context.Context, currentPulse, newPulse c
...
-				"SPLIT HAPPENED parent: %v, left: %v, right: %v",
-				jetID.DebugString(),
-				leftJetID.DebugString(),
-				rightJetID.DebugString(),
-			)
+			logger.WithFields(map[string]interface{}{
+				"parent":      jetID.DebugString(),
+				"left_child":  leftJetID.DebugString(),
+				"right_child": rightJetID.DebugString(),
+			}).Info("split")
```
Comment (nordicdyno) [inline ledger/pulsemanager/pulsemanager.go]: jet split done

File: ledger/storage/genesisinitializer.go [hunk 17]
```diff
@@ -55,8 +54,6 @@ func (gi *genesisInitializer) GenesisRef() *core.RecordRef {
 }
 
 func (gi *genesisInitializer) Init(ctx context.Context) error {
-	inslog := inslogger.FromContext(ctx)
-	inslog.Debug("start storage bootstrap")
```
Comment (nordicdyno) [inline ledger/storage/genesisinitializer.go]: Info

**Pull request merged**